loom/common/assets/assets.cpp

```diff
@@ -1016,15 +1048,17 @@ void *loom_asset_lock(const char *name, unsigned int type, int block)
     // If not loaded, and we aren't ready to block, return NULL.
     if ((block == 0) && (asset->state != loom_asset_t::Loaded))
     {
-        lmLogDebug(gAssetLogGroup, "Not blocking and not loaded yet; lock of '%s' failed.", namePtr);
+        lmLogDebug(gAssetLogGroup, "Unable to lock without blocking, not loaded yet: '%s'", namePtr);
         loom_mutex_unlock(gAssetLock);
         return NULL;
     }
 
+    bool preloaded = asset->state == loom_asset_t::Loaded;
+
     // Otherwise, let's force it to load now.
-    if (asset->state != loom_asset_t::Loaded)
+    if (!preloaded)
     {
-        lmLogDebug(gAssetLogGroup, "Blocking so forcing load of '%s'.", namePtr);
+        lmLogDebug(gAssetLogGroup, "Loading '%s'", namePtr);
```

Comment:
That's really the only place loading happens. If blocking isn't enabled and loading fails, it has a more descriptive error message. I think adding implementation details in the log message where it isn't necessary for differentiation only serves to confuse. If you really want to know what's going on it's pretty easy to just search for `"Loading ` and look at the implementation.